example_5_word_cloud/main_2_nltk.py

```diff
@@ -0,0 +1,73 @@
+"""Speakleash: basic wordcloud (using nltk)"""
+
+import os
+
+from speakleash import Speakleash
```

Comment:
Speaklaesh import should be befor WordCloud:

import matplotlib.pyplot as plt
import spacy
from speakleash import Speakleash
from wordcloud import WordCloud

Reply:
Changed to include suggestions